src/apps/Permissions/ManageRolePanel.js

```diff
@@ -356,6 +356,20 @@ class ManageRolePanel extends React.PureComponent {
             </Button>
           )}
 
+          {(this.state.newRoleManagerValue === ANY_ENTITY ||
+            this.state.assignEntityAddress === ANY_ENTITY) && (
+            <Info
+              mode="warning"
+              css={`
+                margin-top: ${3 * GU}px;
+              `}
+            >
+              Be aware that assigning the permission or manager role to this
```

Comment:
Could we update this copy to be dependent on which one (or both) they've picked?

Reply:
Yep; will implement this along with the other warning messages depending on the address picked.